src/Components/WebAssembly/Build/test/BuildIntegrationTests/PublishIntegrationTest.cs

```diff
@@ -101,6 +101,38 @@ public async Task Publish_InRelease_Works()
             Assert.FileCountEquals(result, 1, publishDirectory, "*", SearchOption.TopDirectoryOnly);
         }
 
+        [Fact]
+        public async Task Publish_WithExistingWebConfig_Works()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("standalone", additionalProjects: new[] { "razorclasslibrary", "LinkBaseToWebRoot" });
+            project.Configuration = "Release";
+
+            var webConfigContents = @"<?xml version='1.0' encoding='utf-8' ?>
+< configuration >
+  < system.web >
+  </ system.web >
+</ configuration >";
+            AddFileToProject(project, "web.config", webConfigContents);
+
+            AddSiblingProjectFileContent(project, @"
+<Target Name='CopyWebConfigOnPublish' AfterTargets='Publish'>
+  <Copy SourceFiles='web.config' DestinationFolder='$(PublishDir)' />
+</Target> ");
```

Comment:
I don't quite get what this is for. Don't our publish targets already take care of copying the web.config file? Why did you need to add this target?